pages/team-structure.js

```diff
@@ -0,0 +1,402 @@
+                name: "Team Hub",
+                leader: {
+                    _id: 8,
+                    name: {
+                        first: "Kevin",
+                        last: "Bai"
+                    }
+                }
+            },{
+                _id: 2,
+                name: "Embedded Software",
+                leader: {
+                    _id: 3,
+                    name: {
+                        first: "Ken",
+                        last: "Livington"
+                    }
+                }
+            },{
+                _id: 3,
+                name: "Website",
+                leader: {
+                    _id: 10,
+                    name: {
+                        first: "Jenny",
+                        last: "Ma"
+                    }
+                }
+            },{
+                _id: 4,
+                name: "POD Systems",
+                leader: {
+                    _id: 11,
+                    name: {
+                        first: "Jenny",
+                        last: "Ma"
+                    }
+                }
+            }
+        ]
+    }
+]
+
+const project_json = [
+    {
+        name: "Electrical Lead", 
+        leader: {
+            _id: 2,
+            name: {
+                first: "Steven",
+                last: "Xiong"
+            }
+        }
+    },{
+        name: "Mechanical Lead", 
+        leader: {
+            _id: 3,
+            name: {
+                first: "Ken",
+                last: "Livington"
+            }
+        }
+    }
+]
+
+// TODO: use this later
+const selectionTiers = [
+    {
+        id: 1,
+        label: "Directors"
+    },
+    {
+        id: 2,
+        label: "Team Leads"
+    },
+    {
+        id: 3,
+        label: "Project Head"
+    },
+    {
+        id: 4,
+        label: "Subordinates"
+    },
+];
+
+const teamHierCustomTheme = {
+    ...theme,
+    breakpoints: ["810px","1090px", "1200px"]
+};
+
+
+// TODO: this component should own the "Selected Member state field"
+const TeamHierParentContainer = styled(Card)`
+    border-radius: 0;
+    overflow-x: hidden;
+
+    flex-grow: 1;
+`;
+
+const HierSubSection = styled(SystemComponent)`
+    overflow: hidden;
+    margin-bottom: ${props => props.theme.space[5]}px;
+`;
+
+// This should be a molecule
+// state : not owned --> the particular level we are at 
+// previous and current selections ie) SW team, team hub
+const TeamStructSideNav = styled(SystemComponent)`
+    background-color: #FFFFFF;
+    margin: ${props => props.theme.space[4]}px;
+`;
+
+class GridListParentContainer extends React.Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            page: 0
+        }
+    }
+
+    onDeterminePage = (e) => {
+        if ((this.state.page+1) * 4 >= this.props.roleCards.length) {
+            this.setState({page: 0})
+        } else {
+            this.setState({page: this.state.page + 1})
+        }
+    }
+
+    render() {
```

Comment:
Currently, the UI is a bit off, the list of members don't span the whole entire width. Instead, could the whole thing be one big horizontal component with horizontal scrolling. The arrow would just help you scroll one way or another. The first time, I didn't actually notice the arrow since the members didn't span all the way.

Reply:
I tried making it span the whole width, but ran into some CSS issues where the member cards' container was crushing and pushing against the side navigation bar and making it skinny, even though I was using "overflow: hidden". I won't be able to get this resolved before the Nov. 30 deadline, but I will write a TODO, and create an identical branch to this which will have this resolved but won't be merged before Nov. 30.